Pipfile

```diff
@@ -14,15 +14,21 @@ gunicorn = "==19.9.0"
 isort = "==4.3.4"
 lazy-object-proxy = "==1.3.1"
 mccabe = "==0.6.1"
-"psycopg2" = "==2.7.5"
-"psycopg2-binary" = "==2.7.5"
+psycopg2 = "==2.7.5"
+psycopg2-binary = "==2.7.5"
 pylint = "==2.1.1"
 six = "==1.11.0"
 whitenoise = "==4.1"
 wrapt = "==1.10.11"
 pytz = "==2018.5"
-Django = "==2.1.1"
```

Comment:
why is this removed?

Reply:
resolved

Reply:
can you set it to `2.1.2` instead? `2.1.1` will give us the Github warning